modules/FindBasisUniversal.cmake

```diff
@@ -188,7 +188,11 @@ foreach(_component ${BasisUniversal_FIND_COMPONENTS})
                     lodepng.cpp
                     # Added in 1.16
                     basisu_opencl.cpp
-                    pvpngreader.cpp)
+                    pvpngreader.cpp
+                    # Added in 1.50
+                    basisu_astc_hdr_enc.cpp
+                    3rdparty/android_astc_decomp.cpp
+                    3rdparty/tinyexr.cpp)
```

Comment:
Haha oh god, tinyexr. At this point I should really start shipping my own dummy stubs for all these...